v1/metrics.yaml

```diff
@@ -49,7 +49,7 @@ paths:
       parameters:
         - name: project
           in: path
-          description: If you want to filter by project, use the domain of any Wikimedia project, for example 'en.wikipedia.org', 'www.mediawiki.org' or 'commons.wikimedia.org'. If you are interested in all pageviews regardless of project, use all-projects.
+          description: The name of any Wikimedia project formatted like {language code}.{project name}, for example en.wikipedia. You may pass en.wikipedia.org and the .org will be stripped off.  For projects like commons without language codes, use commons.wikimedia.  For projects like www.mediawiki.org, you can use that full string, or just use mediawiki or mediawiki.org.
```

Comment:
The domains work, and that's why this documentation explained it the way it was.  This explanation is confusing and I think we should just use the previous version everywhere, especially towards the public.

We store data the way we do for good reason, and we can't hard-code to only allow the domains because of that and special projects like all-projects.  We're close to adding wiki-projects as pseudo-sub-projects here and that will complicate things even more.  So I see no reason to expose the internals of how this works to consumers.